tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -96,6 +96,8 @@ func TestOneLargeFile(t *testing.T) {
 
 		_, err = eng.ExecAction(ctx, engine.RestoreSnapshotActionKey, snapOut)
 		require.NoError(t, err)
+
+		tryDeleteAction(ctx, t, engine.DeleteRandomSnapshotActionKey, nil)
```

Comment:
Placed here so that the latest snapshot can be considered for deletion but the restore does not fail because of it.

Reply:
I'm not sure I understand - why is the delete-snapshot being added to this test in particular? And why only this test?

Reply:
Followup question: where else is the "DeleteRandomSnapshot" action executed?

Reply:
The reason for adding it to this test was to add some kind of delete action to this test, it did not execute any delete actions before this. 
On second thought, I will add the action to all the tests to follow the established pattern.

Reply:
what are the implications of adding a delete action to every test? If we need to "start over" with a fresh repo-under-test, does that mean it'll only keep a single snapshot at steady state?

Asked another way: if every test creates one snapshot then deletes one snapshot (except when there's only one), we'll always be ending the test with only a single snapshot left in the repo, right?

Reply:
Yes, that was the reason behind adding the action to just one test.
I think the way to move forward here is to
1. call delete random sub dir and delete dir content in every test 
2. call delete snapshot action in a couple of tests, not all so that the repo under test can grow in size long term

Reply:
What about putting this action in its own separate test? would that work?

Reply:
done

Reply:
@redgoat650 Does this work for you?
Merging now, we can iterate if needed.